apps/settings/test/marionette/app/regions/device_info.js

```diff
@@ -91,19 +92,20 @@ DeviceInfoPanel.prototype = {
       });
   },
 
-  tapMoreInfoBackBtn: function() {
-    this.waitForElement('moreInfoBackBtn').tap();
-    this.waitForElement('aboutPanel');
-  },
-
   get isDeveloperMenuItemVisible() {
     return this.findElement('developerMenuItem').displayed();
   },
 
   tapDeviceInfoBackBtn: function() {
+    this.client.helper.wait(300);
```

Comment:
I was unable to get it working with waitForElement, and the only solution I could find was introducing a small delay. Please let me know if you think there's a better way. Thanks